### ISSUE TITLE
[Bug] UI - Duplicated Messages

### ISSUE THREAD

**Issue by astra-io** [sgtaziz/WebMessage#128]
Not sure why this happens, happens a bit after message says Delivered. 
Goes away when I refresh or reenter conversation.

![image](https://user-images.githubusercontent.com/26584539/111087624-dc16fe80-84df-11eb-973a-652baf9c7ea6.png)

**Comment by CSab6482** [comment]
[Me too](https://github.com/sgtaziz/WebMessage/issues/115#issuecomment-798615354)

**Comment by shallax** [comment]
Same problem - happens when scrolling up and down the list of chats rather than scrolling within a chat's text for me.

Anything to do with this? Seems to fetch the first 25 messages every time it's called. Also seems to be the last 25 messages that get duplicated for me.

```javascript
cacheMessages () {
      if (!this.$store.state.cacheMessages) return
      if (this.$socket && this.$socket.readyState == 1) {
        for (let i = 0; i < this.chats.length; i++) {
          let chat = this.chats[i]
          this.sendSocket({ action: 'fetchMessages', data: {
              id: chat.personId,
              offset: `0`,
              limit: `25`
            }
          })
        }
      } else {
        setTimeout(this.cacheMessages, 1000)
      }
    },
```
Update: If I turn off message caching in the client options, the problem goes away. Definitely something to do with this.

**Comment by sgtaziz** [comment]
I'll push a quick patch for this issue soon before I finish my current release roadmap. Good find @shallax

**Comment by CSab6482** [comment]
The incidence is much lower than on 0.6.1, but I still get this bug on 0.6.2
![Still broken](https://user-images.githubusercontent.com/53708281/113067603-a1bd8a80-918a-11eb-8a74-c4506f24b3f2.png)
![PNG image](https://user-images.githubusercontent.com/53708281/113067604-a1bd8a80-918a-11eb-9b68-da442b2768c2.png)

**Comment by sgtaziz** [comment]
Fixed with 0.7.0
